support-payment-api/src/main/scala/backend/AmazonPayBackend.scala

```diff
@@ -113,34 +114,16 @@ class AmazonPayBackend(
     }
   }
 
-  private def trackContribution(payment: AuthorizationDetails, acquisitionData: AmazonPayAcquisition, email: String, identityId: Option[Long], clientBrowserInfo: ClientBrowserInfo): EitherT[Future, BackendError, Unit] = {
+  private def trackContribution(payment: AuthorizationDetails, acquisitionData: AmazonPayAcquisition, email: String, identityId: Option[Long], clientBrowserInfo: ClientBrowserInfo): Future[List[BackendError]] = {
```

Comment:
For tracking we only care about the failures, for logging. So we've refactored `trackContribution` in all payment backends to return just a List of errors